benchmarks/attic/bfstree.jl

```diff
@@ -1,69 +0,0 @@
-using LightGraphs
```

Comment:
Most made it into `benchmark/`, but now that we're using `PkgBenchmark`, we need to standardize on the tests.